File: core/src/test/resources/java-samples/SampleExceptions.java
package com.example;

import java.io.IOException;

public class SampleExceptions {
    public void testExceptions() {
        try {
            doSomething();
        } catch (IOException e) { // +1 Exception Handling
            e.printStackTrace();
        } catch (Exception e) { // +1 Exception Handling
            e.printStackTrace();
        } finally { // +1 Exception Handling
            System.out.println("Done");
        }
    }

    private void doSomething() throws IOException {
        throw new IOException("Error");
    }
}

File: languages/kotlin/src/test/resources/kotlin-samples/SampleExceptions.kt
package com.example

class SampleExceptions {
    fun testTryCatch() {
        try { // +1 Exception Handling (try)
            val x = 1 / 0
        } catch (e: ArithmeticException) { // +1 Exception Handling (catch)
            println("Error")
        } finally { // +1 Exception Handling (finally)
            println("Done")
        }
    }
}
